forge-gui/res/cardsfolder/upcoming/nahiris_resolve.txt

```diff
@@ -0,0 +1,11 @@
+Name:Nahiri's Resolve
+ManaCost:3 R W
+Types:Enchantment
+S:Mode$ Continuous | Affected$ Creature.YouCtrl | AddPower$ 1 | AddKeyword$ Haste | Description$ Creatures you control get +1/+0 and have haste.
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigChangeZone | TriggerDescription$ At the beginning of your end step, exile any number of nontoken artifacts and/or creatures you control. Return those cards to the battlefield under their owner's control at the beginning of your next upkeep.
+SVar:TrigChangeZone:DB$ ChangeZone | MinChoiceAmount$ 0 | ChoiceAmount$ Y | Choices$ Creature.YouCtrl+nonToken,Artifact.YouCtrl+nonToken | ChoiceTitle$ Choose any number of nontoken artifacts and creatures you control | Origin$ Battlefield | Destination$ Exile | SubAbility$ DelTrig | RememberChanged$ True
```

Comment:
Wild mixing: these Choice params don't exist

Reply:
Yeah mixed up the choice params of putcounter